source/wp-content/themes/wporg-developer-2023/package.json

```diff
@@ -16,8 +16,8 @@
 		"extends": "../../../../.stylelintrc"
 	},
 	"scripts": {
-		"build": "wp-scripts build",
-		"start": "wp-scripts start",
+		"build": "wp-scripts build --experimental-modules",
+		"start": "wp-scripts start --experimental-modules",
```

Comment:
I could pull this out to its own PR to upgrade the package and add this flag. I don't think it should affect anything else.